tests/momento/requests/vector_index/test_item.py

```diff
@@ -5,22 +5,45 @@
 from momento.requests.vector_index import Item
 
 
-def test_serialize_item_with_string_metadata() -> None:
-    item = Item(id="id", vector=[1, 2, 3], metadata={"key": "value"}).to_proto()
+def test_serialize_item_with_diverse_metadata() -> None:
+    item = Item(
+        id="id",
+        vector=[1, 2, 3],
+        metadata={
+            "string_key": "value",
+            "int_key": 1,
+            "double_key": 3.14,
+            "bool_key": True,
+            "list_of_strings_key": ["one", "two"],
+        },
+    ).to_proto()
```

Comment:
Just for my knowledge - is there an advantge in converting back to `proto` to test fields?

Reply:
I wanted to specifically test we were converting to protobuf correctly, as I've had typos/copypasta errors in the past with oneofs.